ast.go

```diff
@@ -144,6 +144,7 @@ func (g *graph) hydrateFile(pkg Package, f *descriptor.FileDescriptorProto) File
 	}
 	g.add(fl)
 
+	fl.fileDependencies = make([]File, 0)
```

Comment:
Can we just leave this as unset (so delete the line)? Appending to a nil slice is valid

```suggestion
```